backend/src/main/kotlin/org/loculus/backend/service/submission/UpdateTrackerTable.kt

```diff
@@ -0,0 +1,8 @@
+import org.jetbrains.exposed.sql.Table
+
+const val UPDATE_TRACKER_TABLE_NAME = "table_update_tracker"
+
+object UpdateTrackerTable : Table(UPDATE_TRACKER_TABLE_NAME) {
+    val lastTimeUpdatedDbColumn = varchar("last_time_updated", 255).nullable()
+    val tableNameColumn = varchar("table_name", 255)
```

Comment:
This doesn't match the declaration in the SQL file.
```suggestion
    val tableNameColumn = text("table_name")
    val lastTimeUpdatedDbColumn = datetime("last_time_updated")

    override val primaryKey = PrimaryKey(tableNameColumn)
```

Reply:
That's on purpose, whatever comes out of the db is a blackbox ordered string (ascending) - that's all we need, no conversion necessary, any conversion may introduce bugs

Reply:
Then let's make it a `text`.

Reply:
But I'm not so sure about it. The Etag should be a comparable, but otherwise black-box string. At this point, we can still be specific about the type. Maybe this table will be used in another context later, where it's actually better to have the actual type.